### PR TITLE
Fix new disk proposal settings

### PR DESCRIPTION
**Pull request by dzedro**
https://github.com/yast/yast-storage/pull/200

### CODE REVIEW

File: tests/installation/partitioning_lvm.pm [hunk 1]
```diff
@@ -16,23 +16,26 @@ use testapi;
...
-        type_password;
-        send_key "tab";
-        type_password;
-        send_key "ret", 1;
+        send_key "alt-e";    # enable encrypted LVN-based proposal
+    }
+    else {
+        send_key "alt-l";    # enable LVM-based proposal
+    }
+    assert_screen "inst-encrypt-password-prompt";
```
Comment (sysrich) [inline tests/installation/partitioning_lvm.pm]: This will not work - you only get asked for a password prompt if using Encryption

Comment (dzedro) [inline tests/installation/partitioning_lvm.pm]: after encryption is picked password popup will come up same as before, I will test it.
https://openqa.suse.de/tests/286092/modules/partitioning_lvm/steps/1/edit

Comment (sysrich) [inline tests/installation/partitioning_lvm.pm]: You're right, but the code you have written will try and assert a password prompt and enter a password even when get_var("ENCRYPT") is not set

Comment (dzedro) [inline tests/installation/partitioning_lvm.pm]: true

**Comment by sysrich** [comment]
Need to do a proper solution that will work with openSUSE and maintenance updates..watch this space

**Pull request closed without merging**